src/data/startups.yml

```diff
@@ -91,6 +91,17 @@
       subTitle: code source
       href: https://github.com/SocialGouv/fce
       description: Bientôt ! Le code source de l'application FCE
+- id: MANO
```

Comment:
```suggestion
- id: mano
```

Reply:
should be lowercase as referenced here : `<LayoutArticle meta={meta} startup="mano">`